parlai/tasks/dream/agents.py

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Facebook, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+from parlai.core.teachers import FixedDialogTeacher
+from .build import build
+import os
+import json
+
+
+def setup_data(opt, jsons_path):
+    if opt['datatype'].startswith('test'):
+        dpath = os.path.join(jsons_path, 'test.json')
+    elif opt['datatype'].startswith('valid'):
+        dpath = os.path.join(jsons_path, 'dev.json')
+    else:
+        dpath = os.path.join(jsons_path, 'train.json')
+    episodes = []
+    with open(dpath) as f:
+        data = json.load(f)
+        for dialogue in data:
+            context = '\n'.join(dialogue[0])
+            qas = dialogue[1]
+            episodes.append({'context': context, 'qas': qas})
+    return episodes
+
+
+def num_examples(episodes):
+    examples = 0
+    for data in episodes:
+        examples += len(data['qas'])
+    return examples
+
+
+def get(tid, episodes, episode_idx, entry_idx=0):
```

Comment:
hmm - it looks like the arguments here don't line up with the function call on line 69